test/v2/ThirdPartyRegistry.spec.js

```diff
@@ -2289,17 +2289,26 @@ describe('ThirdPartyRegistry', function () {
     it('reverts when oracle.getRate attempts to change the state', async function () {
       const oracleContract = await InvalidOracle.new()
 
-      thirdPartyRegistryContract = await ThirdPartyRegistry.new(
-        owner,
-        thirdPartyAgregator,
-        collector,
-        committeeContract.address,
-        manaContract.address,
-        oracleContract.address,
-        oneEther,
+      const ThirdPartyRegistryFactory = await ethers.getContractFactory(
+        'ThirdPartyRegistry'
+      )
+
+      const proxy = await upgrades.deployProxy(
+        ThirdPartyRegistryFactory,
+        [
+          owner,
+          thirdPartyAgregator,
+          collector,
+          committeeContract.address,
+          manaContract.address,
+          oracleContract.address,
+          oneEther.toString(),
+        ],
         fromDeployer
       )
 
+      thirdPartyRegistryContract = await ThirdPartyRegistry.at(proxy.address)
```

Comment:
As deployProxy returns the contract instance with the ethers interface, I'm using that deployed contract address wrapped in web3 so no changes to the tests have to be made. Thanks @nachomazzara for this.